week-2/L-array-methods-2/exercise2.js

```diff
@@ -23,3 +25,4 @@ console.log(capitalise("hello"));
   Daniel
   Hello
 */
+/* Nate i didnt get this practice well , please explain me about it .*/
```

Comment:
The trick is to convert the string into an array and then put it back into a string again. You can use the `String.split()` function to turn a string into an array, like this:

```
var name = "Nate";
var nameArray = name.split();
console.log(nameArray); // Outputs: ["N", "a", "t", "e"]
```

You can then put the name back together with `String.join()`:

```
name = nameArray.join();
console.log(name); // Outputs: "Nate"
```

Once you've got your name into an array, how would you capitalise the first letter before converting the name back into a string?